Vital/Controllers/AccountController.cs

```diff
@@ -63,16 +65,53 @@ public async Task<IActionResult> ResetPassword([FromBody] ResetPasswordDto dto)
 
         var user = await _userManager.FindByIdAsync(dto.UserId.ToString());
 
-        if (user is not null)
+        if (user is null)
+        {
+            throw new ResetPasswordException("Something went wrong.");
```

Comment:
i think we could just return here, instead of throwing an exception

Reply:
i don't think i get it